src/main/tv/codely/scala_http_api/module/course/domain/Course.scala

```diff
@@ -1,14 +1,15 @@
 package tv.codely.scala_http_api.module.course.domain
 
+import tv.codely.scala_http_api.module.shared.course.domain.CourseId
 import tv.codely.scala_http_api.module.shared.user.domain.UserId
 
 object Course {
-  def apply(id: String, title: String, lessons: BigDecimal, creatorId: String): Course = Course(
+  def apply(id: String, title: String, totalLessons: Int, creatorId: String): Course = Course(
     CourseId(id),
     CourseTitle(title),
-    CourseLessons(lessons),
+    CourseLessons(totalLessons),
```

Comment:
Now we've a mismatch between the attribute name and its class. It would be awesome to rename `CourseLessons` to `CourseTotalLessons` in order to avoid this mismatch 😬 🙂